### PR TITLE
PackAsTool detection

### PR DESCRIPTION
**Pull request by arturcic**
Implements #87

### CODE REVIEW

File: src/Cake.Incubator/ProjectParserExtensions.cs [hunk 1]
```diff
@@ -42,6 +42,23 @@ public static bool IsLibrary(this CustomProjectParserResult projectParserResult)
             return projectParserResult.OutputType.EqualsIgnoreCase("Library");
         }
 
+        /// <summary>
+        /// Checks if the project is a library
+        /// </summary>
+        /// <param name="projectParserResult">the parsed project</param>
```
Comment (wwwlicious) [inline src/Cake.Incubator/ProjectParserExtensions.cs]: correct docs for this method

Comment (arturcic) [reply src/Cake.Incubator/ProjectParserExtensions.cs]: sure,

Comment (arturcic) [reply src/Cake.Incubator/ProjectParserExtensions.cs]: done

**Comment by gep13** [comment]
@arturcic if we can get the build error sorted, then you should be able to consume the package from here:

https://ci.appveyor.com/nuget/cake-incubator

Similar to how I am currently fetching the latest Cake.Recipe version in this build here:

https://github.com/GitTools/GitReleaseManager/blob/develop/recipe.cake#L1

**Pull request merged**